client/scss/components/_gif_video.scss

```diff
@@ -6,23 +6,14 @@
   background: transparent;
   border: 0;
   appearance: none;
-  top: 10px;
+  top: $spacing-unit;
   left: 10px;
```

Comment:
Why `10` and `spacing-unit`?

Reply:
Good question – best I've got is, 'it looked better to me', but I imagine this play/pause button could use a bit of UX input anyhow. Maybe @Heesoomoon can help.